CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Fix PLP scroll bug after applying filters for the mobile version (#454)
+- Fixed BaseStore logo right margin on mobile devices (#455)
```

Comment:
Minor suggestion so [the last version comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first).
```suggestion
- Fixed BaseStore logo right margin on mobile devices (#455)
- Fix PLP scroll bug after applying filters for the mobile version (#454)
```

Reply:
Nice.